book/src/drivers.md

```diff
@@ -49,6 +49,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [IBM Spectrum Scale](https://github.com/IBM/ibm-spectrum-scale-csi) | `spectrumscale.csi.ibm.com` | v1.0, v1.1 | A Container Storage Interface (CSI) [Driver](https://www.ibm.com/support/knowledgecenter/STXKQY_CSI_SHR) for the IBM Spectrum Scale File System | Persistent | Read/Write Multiple Pod | Yes |
 [Infinidat](https://github.com/Infinidat/infinibox-csi-driver) | `infinibox-csi-driver` | v1.0, v1.1 | A Container Storage Interface (CSI) Driver for Infinidat [InfiniBox](https://infinidat.com/en/products-technology/infinibox) | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
 [Inspur InStorage CSI](https://github.com/OpenInspur/instorage-k8s) | `csi-instorage` | v1.0 | A Container Storage Interface (CSI) Driver for Inspur AS/HF/CS/CF Series Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[Inspur InStorage CSI for Mass Storage](https://github.com/OpenInspur/instorage-k8s) | `csi-instorage` | v1.0 | A Container Storage Interface (CSI) Driver for Inspur Mass Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, NFS, Snapshot, Expansion, Cloning
```

Comment:
It seems that the instructions on how to use this CSI driver is only in Chinese?  I suggest that you also add instructions in English.